### DISCUSSION TITLE
QTabs doesn't slide when its container has RTL direction

### DISCUSSION THREAD

**Discussion by saifobeidat** [quasarframework/quasar#10928] [General - Components / Directives / etc]
Hello, 

I faced a weird thing where QTabs component doesn't work probably when its container div has RTL direction. it doesn't not slide the items.

If you notice below: the container contains a property `dir="rtl"`  if I remove it, it will work as expected.
```
<div id="q-app">
  <div class="q-pa-md" dir="rtl">
    <div class="q-gutter-y-md" style="max-width: 300px">
      <q-tabs
        v-model="tab"
        inline-label
        outside-arrows
        mobile-arrows
        class="bg-primary text-white shadow-2"
      >
        <q-tab name="mails" icon="mail" label="Mails"></q-tab>
        <q-tab name="alarms" icon="alarm" label="Alarms"></q-tab>
        <q-tab name="movies" icon="movie" label="Movies"></q-tab>
        <q-tab name="photos" icon="photo" label="Photos"></q-tab>
        <q-tab name="videos" icon="slow_motion_video" label="Videos"></q-tab>
        <q-tab name="addressbook" icon="people" label="Address Book"></q-tab>
      </q-tabs>

    </div>
  </div>
</div>
```

anyway or workaround to fix this?

Here is a codepen for the issue:
https://codepen.io/saifobiedat/pen/jOLNwoJ?editors=101

**Comment by ontwikkelfabriek** [comment]
@saifobeidat 

This looks like a bug to me. Could you create a Github issue for this?

**Comment by pdanpdan** [comment]
ref #8578

**Comment by pdanpdan** [comment]
Btw, it's not a good idea to use `dir` on any other element except on  `html` - the checks for RTL/LTR in CSS are done considering only one direction for the whole document, and you will end up in some bad situations when both RTL and LTR styles will be applied.
